lib/mongo/collection/view/iterable.rb

```diff
@@ -39,7 +39,7 @@ def each
           @cursor = select_cursor(session)
 
           if QueryCache.enabled? && @cursor.is_a?(Mongo::CachingCursor)
-            QueryCache.cache_table[cache_key] = @cursor
+            QueryCache.write(@cursor, cache_options)
```

Comment:
What do you think about get/set instead of read/write?

Reply:
I'm totally happy with that!